src/Sinch/Core/Utils.cs

```diff
@@ -144,4 +189,10 @@ public static void ThrowNullDeserialization(Type type)
             throw new InvalidOperationException($"{type.Name} deserialization result is null");
         }
     }
+
+    internal enum PageStart
+    {
+        Zero = 0,
+        First = 1,
```

Comment:
The naming is misleading as we are looking for the first page index in all cases. I think "One" would be be more understandable. Or `IndexZero` and `IndexOne`